modules/luci-mod-system/htdocs/luci-static/resources/view/system/system.js

```diff
@@ -49,16 +49,17 @@ callTimezone = rpc.declare({
 });
 
 function formatTime(epoch) {
-	var date = new Date(epoch * 1000);
-
-	return '%04d-%02d-%02d %02d:%02d:%02d'.format(
-		date.getUTCFullYear(),
-		date.getUTCMonth() + 1,
-		date.getUTCDate(),
-		date.getUTCHours(),
-		date.getUTCMinutes(),
-		date.getUTCSeconds()
-	);
+	var date = new Date(epoch * 1000),
+		zn = uci.get('system', '@system[0]', 'zonename')?.replaceAll(' ', '_') || 'UTC',
+		ts = uci.get('system', '@system[0]', 'timestyle'),
+		hc = uci.get('system', '@system[0]', 'hourcycle');
```

Comment:
While we're here, let's give these variable names some more global context for users to identify them in the uci config files. Like `clock_time_style` and `clock_hourcycle`.